utils/cronjobs_osgeo_lxd/cron_job_list_grass

```diff
@@ -29,9 +29,13 @@
 50 02 * * 6 nice sh /home/neteler/cronjobs/cron_grass7_relbranch_src_snapshot.sh
 
 # daily Linux binary snapshots, also creates main manuals, addon manuals, and prog-manual (target dir: /var/www/code_and_data/)
-05 05 * * * nice sh /home/neteler/cronjobs/cron_grass8_relbranch_build_binaries.sh > /var/www/code_and_data/grass80/binary/linux/snapshot/build.log.txt 2>&1
-05 06 * * * nice sh /home/neteler/cronjobs/cron_grass7_relbranch_build_binaries.sh > /var/www/code_and_data/grass78/binary/linux/snapshot/build.log.txt 2>&1
-05 07 * * * nice sh /home/neteler/cronjobs/cron_grass8_main_build_binaries.sh > /var/www/code_and_data/grass81/binary/linux/snapshot/build.log.txt 2>&1
+# old stable: G78
+05 05 * * * nice sh /home/neteler/cronjobs/cron_grass7_relbranch_build_binaries.sh > /var/www/code_and_data/grass78/binary/linux/snapshot/build.log.txt 2>&1
+# stable: G82
+05 15 * * * nice sh /home/neteler/cronjobs/cron_grass8_relbranch_build_binaries.sh > /var/www/code_and_data/grass82/binary/linux/snapshot/build.log.txt 2>&1
+# dev: G83
```

Comment:
Actually, can you remove the version numbers from here? They are not used. The version is determined in the script. Less places to edit.

(I'm adding this to the release procedure.)